completion/ksmisc.cwl

```diff
@@ -0,0 +1,13 @@
+# ksmisc package
+# ponte-vecchio 2023-01-17
+
+#include:ksjosaref
+#include:kstextks
+#include:kslinematters
+#include:preparefont
+#include:singlepageempty
+
+\ks_pop_left:NN %<tlvar1%> %<tlvar2%>#S
+\ks_pop_right:NN %<tlvar1%> %<tlvar2%>#S
+\ks_clist_set_from_tl:NN %<clistvar%> %<tlvar%>#S
+\ks_seq_set_from_tl:NN %<seqvar%> %<tlvar%>#S
```

Comment:
There's not currently any support for expl3 syntax so these lines don't do anything

Reply:
Removed!